website/static/js/nodesDelete.js

```diff
@@ -248,7 +279,7 @@ NodesDeleteViewModel.prototype.confirmChanges =  function() {
     if ($('#bbConfirmTextDelete').val() === this.confirmationString) {
         if (nodesChanged.length <= 100) {
             $osf.block('Deleting Project');
-            batchNodesDelete(nodesChanged.reverse()).then(function () {
+            batchNodesDelete(nodesChanged.reverse(), self.nodeType).then(function () {
                 self.page(self.WARNING);
             }).fail(function (xhr) {
                 $osf.unblock();
```

Comment:
[L290](https://github.com/CenterForOpenScience/osf.io/pull/7911/files#diff-de94aefe74754843ec4d7c25c5b9275bR290) below, Growl message says "Problem deleting project" regardless of whether this is a project or component.